camayoc/tests/qpc/api/v1/utils.py

```diff
@@ -12,6 +13,108 @@
     FailedScanException,
     WaitTimeError,
 )
+from camayoc.qpc_models import (
+    Credential,
+    Scan,
+    Source
+)
+
+
+def get_source(source_type, cleanup):
+    """Retrieve a single network source if available from config file.
+
+    Retreive one network source from the config file.
+    Sources listed under the following section are assumed to be available
+    on demand and do not require their power state to be managed.
+
+    The expected configuration in the Camayoc's configuration file is as
+    follows::
+
+        qpc:
+        # other needed qpc config data
+            sources:
+                 - hosts:
+                       - '127.0.0.1'
+                   name: local
+                   type: network
+                   credentials:
+                       - root
+
+    The credential listed is assumed to be in the top level 'credentials'
+    section.
+
+    This source is meant to be used for tests where we do not care about the
```

Comment:
By `This source` you mean the config file source or the source returned by this function? I think the sources under qpc->sources can be used for scan too.